src/pages/home/report/ReportActionCompose/ComposerWithSuggestions.js

```diff
@@ -128,6 +129,12 @@ function ComposerWithSuggestions({
     // A flag to indicate whether the onScroll callback is likely triggered by a layout change (caused by text change) or not
     const isScrollLikelyLayoutTriggered = useRef(false);
 
+    const hasEnoughSpaceForLargeSuggestion =
+        listHeight - composerHeight - (CONST.CHAT_FOOTER_SECONDARY_ROW_HEIGHT + 2 * CONST.CHAT_FOOTER_SECONDARY_ROW_PADDING) >
```

Comment:
Maybe we can extract some local `const`s for components of this expression? This is _big_.